.github/workflows/check-source-signatures.yml

```diff
@@ -72,44 +83,27 @@ jobs:
 
       - name: Check for invalid source signatures
         run: |
-          if [ -z "${{ env.changed_pkgs }}" ] && [ -z "${{ env.changed_pkgs_extended }}" ]; then
-            echo "No package changes detected."
+          if [ -z "${{ env.changed_pkgs }}" ]; then
+            echo "No package changes detected in '${{ matrix.specs-dir }}''."
             exit 0
           fi
 
-          # Core SPECs
           if [ -n "${{ env.changed_pkgs }}" ]; then
-            # We want to ignore errors here, as we want to check all the packages that have been modified. Capture the error code and check it later.
-            set +e
-            set -x
-            sudo make -C toolkit -j$(nproc) input-srpms REBUILD_TOOLS=y DAILY_BUILD_ID=${{ env.LKG_BUILD_ID }} SRPM_PACK_LIST="${{ env.changed_pkgs }}"
-            core_err=$?
-            set +x
-            set -e
-          fi
-
-          # Extended SPECs
-          if [ -n "${{ env.changed_pkgs_extended }}" ]; then
-            # We want to ignore errors here, as we want to check all the packages that have been modified. Capture the error code and check it later.
+            # We want to ignore errors here so we can print a helpful message at the end.
```

Comment:
Idea: if you want to drop disabling/enabling `-e` and the `err` variable, you can also go for:
```bash
if ! sudo make -C toolkit (...); then
  printf "\n\n******************************"
...
```

Reply:
I think the `set -x` will produce messy output if its outside the conditional, I'll probably leave it like this.

Reply:
Moved to using the conditional since the output wasn't formatting nicely the other way